exporter/exporterhelper/internal/producer_consumer_queue.go

```diff
@@ -19,4 +32,8 @@ type ProducerConsumerQueue interface {
 	// Stop stops all consumers, as well as the length reporter if started,
 	// and releases the items channel. It blocks until all consumers have stopped.
 	Stop()
+	// Capacity returns the capacity of the queue.
+	Capacity() int
+	// IsPersistent returns true if the queue is persistent.
+	IsPersistent() bool
```

Comment:
Add a TODO to not expose this public if the struct moves to a public package.

Reply:
added